### DISCUSSION TITLE
I am getting error 500 in a particular table (not in all table) and no data visible in this table at dashboard. But I am able to access data in my web app.

### DISCUSSION THREAD

**Discussion by roker15** [supabase/supabase#3847] [Questions] [answered]
I am getting error 500 in a particular table (not in all table) and no data visible in this table at dashboard. But I am able to access data in my web app. Here is the image--
![supabase image](https://user-images.githubusercontent.com/59526869/141047561-c3caf1aa-d548-4ee1-8e7f-7c9e99ec2790.PNG)
--

**Comment by inian** [comment]
Hi please create a support ticket here https://app.supabase.io/support/new

**Reply by roker15** [reply]
yes here is screen shot
![ww](https://user-images.githubusercontent.com/59526869/141513384-b4e915f4-2cfe-44cd-8aa4-2070f7680d6d.png)

![ss](https://user-images.githubusercontent.com/59526869/141513664-6949676a-747f-4bd2-ab73-67e6cd497efd.png)

**Comment by darora** [comment, marked as answer]
For posterity, this was being caused by the freetext columns resulting in some massive rows, that caused the maximum response payload size to be too large. We'll be looking into improving the error message shown under these circumstances.
